requirements.txt

```diff
@@ -3,3 +3,6 @@ git+https://github.com/CodeForAfrica/HURUmap.git@0.0.7#egg=hurumap
 # Workaround for dependency links
```

Comment:
@KhadijaMahanga, aren't we removing this file for now since Wazimap/HURUmap already has the API support built in?

Reply:
@kilemensi yes we are!